src/structures/User.js

```diff
@@ -98,6 +98,16 @@ class User {
   }
 
   /**
+   * Get the note set for the user
+   * <warn>This is only available for user accounts.</warn>
+   * @type {?string}
+   * @readonly
+   */
+  get note() {
+    return this.client.user.notes.get(this.id);
```

Comment:
Add an `|| null` to the end here, so that `undefined` doesn't get returned if there's no note.